kube_dns/metadata.csv

```diff
@@ -0,0 +1,8 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+kubedns.response_size.bytes.sum,gauge,,response,byte,Size of the returns response in bytes.,0,kube_dns,response size sum
+kubedns.response_size.bytes.count,gauge,,response,,Number of requests on which the kubedns.response_size.bytes.sum metric is evaluated.,0,kube_dns,response size sample count
```

Comment:
Technically it's requests for which a response was sent so it's response in the unit. Updating the description. Good catch! :)